src/depends.py

```diff
@@ -105,10 +128,11 @@ def check_openssl():
     try:
```

Comment:
Same here..

  try:
         import ctypes
     except ImportError:
        logger.error(
            'Unable to check OpenSSL. The ctypes module is not available.')
         return False